File: SharedComponents/Demo/JSInteropSamples/JSToStaticNET.razor.js
export async function setMessage() {
    const { getAssemblyExports } = await globalThis.getDotnetRuntime(0);
    var exports = await getAssemblyExports("SharedComponents.dll");
    alert(exports.SharedComponents.Demo.JSInteropSamples.JSToStaticNET.GetMessageFromNET());
}

export async function showMessage() {
    await setMessage();
}

File: SharedComponents/Demo/JSInteropSamples/JSToStaticNET.razor.cs
using System.Runtime.InteropServices.JavaScript;
using System.Runtime.Versioning;

namespace SharedComponents.Demo.JSInteropSamples
{
    [SupportedOSPlatform("browser")]
    public partial class JSToStaticNET
    {
        [JSExport]
        internal static string GetMessageFromNET()
        {
            return ".NET";
        }

        [JSImport("showMessage", "jstonet")]
        internal static partial void ShowMessage();
    }
}
